blueocean-git-pipeline/src/main/java/io/jenkins/blueocean/blueocean_git_pipeline/GitPipelineCreateRequest.java

```diff
@@ -40,6 +42,10 @@ public GitPipelineCreateRequest(String name, BlueScmConfig scmConfig) {
 
     @Override
     public BluePipeline create(Reachable parent) throws IOException {
+        User authenticatedUser =  User.current();
+        if(authenticatedUser == null){
+            throw new ServiceException.UnauthorizedException("Must login to create a pipeline");
+        }
```

Comment:
Good point. I will open another ticket to get it addressed properly as it may need some UX change.